types/sortablejs/index.d.ts

```diff
@@ -137,6 +147,15 @@ declare namespace Sortable {
          * time in milliseconds to define when the sorting should start
          */
         delay?: number;
+        /**
+         * Only delay if user is using touch
+         */
+        delayOnTouchOnly?: boolean;
+        /**
+         * Direction of Sortable
+         * (will be detected automatically if not given)
+         */
+        direction?: 'horizontal' | 'vertical';
```

Comment:
This isn't right, `direction` can also be a function. Example in the docs:

```javascript
Sortable.create(el, {
	direction: function(evt, target, dragEl) {
		if (target !== null && target.className.includes('half-column') && dragEl.className.includes('half-column')) {
			return 'horizontal';
		}
		return 'vertical';
	}
});
```

Reply:
Ok, I will add this in.